src/Components/ContentDelivery/ContentDelivery.module.scss

```diff
@@ -37,22 +42,41 @@ h2 {
     }
 }
 
+//modal
+.workMsgModal {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0; 
+  width: 100%;
+  height: 100%; 
+  text-align: center;
+}
+
+.modalContent {
+  background-color: $lightBlueAccent;
+  margin: 20% auto;
+  padding: 1em; 
+  border: 1px solid $midBlue;
+  width: 30%; 
+}
+
 @media screen and (max-width: 992px) {
-  .video-wrapper {
+  .videoWrapper {
     @include size(42em, 24em);
   }
 
-  .video-player {
+  .videoPlayer {
     padding: 1.5em;
   }
 }
 
 @media screen and (max-width: 688px) {
-  .video-wrapper {
+  .videoWrapper {
     @include size(26em, 15em);
   }
 
-  .video-player {
+  .videoPlayer {
     padding: 0.5em;
   }
 }
```

Comment:
This stylesheet is super fly, way to rock the includes and mixins and that blur!